aspnetcore/grpc/authn-and-authz.md

```diff
@@ -171,6 +171,43 @@ The preceeding code:
 * Registers the `GreeterClient` type with client factory.
 * Configures the `AuthInterceptor` for this client using `InterceptorScope.Client`. A new interceptor is created for each client instance. When a client is created for a gRPC service or Web API controller, the scoped `ITokenProvider` is injected into the interceptor.
 
+##### Bearer token with gRPC client factory using AsyncAuthInterceptor
```

Comment:
I'll move the final version of this sample if there is one approved by JamesNK to a full client factory app sample for .NET 6 and then pull in the relevant snippets